### ISSUE TITLE
Hunter trap visibility 

### ISSUE THREAD

**Issue by Johanes022** [dalaranwow/dalaran-wow#4842]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Hunter traps are currently visible to all enemy players wich makes traps easly avoidable.

**Expected behaviour**: Traps should not be visible

**Steps to reproduce the problem**:

1. duel a hunter and move away from him
2. watch hunter put down the trap
3. notice that you can see traps from far away

**Include proofs for this behaviour**

https://www.youtube.com/watch?v=tZdX098BcnA

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://wowgaming.altervista.org/aowow/?spell=13809
https://wowgaming.altervista.org/aowow/?spell=34600
for example + the rest of the traps

**Include Screenshots from the issue if necessary**
https://image.prntscr.com/image/qYQa_BoNTTywpRR1bmI9tQ.jpeg

Link that might help
https://github.com/TrinityCore/TrinityCore/issues/18251

**Comment by Coztume** [comment]
#3026

**Comment by Rushor** [comment]
Your issue posted here is a duplicate of issue: #3026 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
